packages/backend/src/controllers/payment.ts

```diff
@@ -24,17 +24,18 @@ export class PaymentCtrl extends RESTController<Payment> {
             transactionalEntityManager,
             membership
           );
-          if (!newRequest) throw new Error();
+          if (!newRequest) throw new Error('Payment request not created');
           const newPayment = await this.createPayment(
             transactionalEntityManager,
             newRequest,
             membership
           );
-          if (!newPayment) throw new Error();
+          if (!newPayment) throw new Error('Payment not created');
```

Comment:
It's better if you throw an "APIError" so you can also send a better code than 500